chromadb/api/types.py

```diff
@@ -313,7 +313,9 @@ def validate_update_metadata(metadata: UpdateMetadata) -> UpdateMetadata:
 def validate_metadatas(metadatas: Metadatas) -> Metadatas:
     """Validates metadatas to ensure it is a list of dictionaries of strings to strings, ints, floats or bools"""
     if not isinstance(metadatas, list):
-        raise ValueError(f"Expected metadatas to be a list, got {metadatas}")
+        raise ValueError(
```

Comment:
Have a look here for how to get better type names - https://github.com/chroma-core/chroma/pull/2254/files